src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  height: 16px;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+
+    &:not(:first-child) {
+      margin-left: 4px;
+    }
+  }
+}
+
+.stars--4 :nth-child(-n+4) {
```

Comment:
The active star selector should be nested inside the .stars block and use the '&' symbol for better readability and maintainability.